deploy/charts/values.yaml

```diff
@@ -94,15 +94,31 @@ kube-prometheus-stack:
 
     config:
       global:
+        smtp_smarthost: 'smtp.org:587'  
```

Comment:
why are we setting these, shouldn't they be set by users?

Reply:
Shall i keep them blank? We will ask user to set them. I think user might have to create a new secret altogether if they want additional receiver like slack, teams etc

Reply:
Agree with @tiagolobocastro here. Either you can leave blank or set examples under comments.

Reply:
`smtp_smarthost`, `to` and `smtp_from` cant be blank. Alertmanager wont start. Others which can be, keeping them unset.

Reply:
I must be missing something here.
Then how is this working otherwise, users set this according to their config? In which case we don't need to set them to anything, users will set it.

Reply:
We wanted to give a child route to match `openebs` alerts. I had used `email_config` as an example receiver. Hence needed some smtp config to be set.  I can also keep it null. Without giving any example prefix.